src/core/account_manager.ts

```diff
@@ -214,25 +214,19 @@ export class AccountManager {
    * @returns a node client that can be used to call transactions
    */
   async _getNodeClient (namespace: string, networkNodeServicesMap: Map<string, NetworkNodeServices>, operatorId: string,
-    operatorKey: string) {
-    const nodes = {}
+    operatorKey: string, useFirstNodeOnly = true) {
```

Comment:
what happens when set useFirstNodeOnly, and first node is target for `solo node delete --node-aliase node1`

Reply:
I think it should be fine.  We rebuild the node client after the network is frozen.

Reply:
bad stuff, I have a change in flight to use the second node only.

Reply:
but, what if they decide to delete the second node and keep the first?